src/quo/components/utilities/token/loader.cljs

```diff
@@ -6,9 +6,10 @@
 
 (defn- get-token-image*
   [token]
-  (let [token-symbol (cond-> token
-                       (keyword? token) name
-                       :always          (comp string/lower-case str))]
-    (get tokens token-symbol)))
+  (when token
```

Comment:
@ulisesmac and I revisted this as we realised it was breaking the token component 👍